test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -269,6 +270,22 @@ public void NgramWorkout()
             Done();
         }
 
+        [Fact]
+        void TestNgramCompatColumns()
+        {
+            string dropModelPath = GetDataPath("backcompat/ngram.zip");
+            string sentimentDataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
+            var data = TextLoader.CreateReader(ML, ctx => (
+                    Sentiment: ctx.LoadBool(0),
+                    SentimentText: ctx.LoadText(1)), hasHeader: true)
+                .Read(sentimentDataPath);
+            using (FileStream fs = File.OpenRead(dropModelPath))
+            {
+                var result = ModelFileUtils.LoadTransforms(Env, data.AsDynamic, fs);
+                var foundColumnFeature = result.Schema.TryGetColumnIndex("Features", out int featureIdx);
```

Comment:
>foundColumnFeature [](start = 20, length = 18)

Were you intending to assert this somewhere? Also, could you please not use the `ISchema` derived functions, but the newer `Schema` ones? We're going to be getting rid of the `ISchema` ones sooner or later. In this case, preferred would be `GetColumnOrNull`.

[https://github.com/dotnet/machinelearning/blob/7e21afa4612ddca53bbf9d67b84df2b58d4636c2/src/Microsoft.ML.Core/Data/Schema.cs#L64](https://github.com/dotnet/machinelearning/blob/7e21afa4612ddca53bbf9d67b84df2b58d4636c2/src/Microsoft.ML.Core/Data/Schema.cs#L64)